src/app/core/redux/app/types.ts

```diff
@@ -0,0 +1,18 @@
+export const SET_LOADING = "SET_LOADING";
+export const SET_AUTHENTICATED = "SET_AUTHENTICATED";
+
+export interface SetLoading {
```

Comment:
Interface names should be Noun. This here is a verb. Can we change this to `SetLoadingPayload` or something?

Reply:
Made changes.